lib/Wikia/src/Service/Helios/HeliosClientImpl.php

```diff
@@ -8,15 +7,26 @@
  *
  * This is a naive implementation.
  */
-class Client
+class HeliosClientImpl implements HeliosClient
```

Comment:
If we change interface name we could just name this class as `HeliosClient` and it will be in consistency with `AuthService` etc.